kawin/Thermodynamics.py

```diff
@@ -49,20 +49,26 @@ class GeneralThermodynamics:
     gOffset = 1      #Small value to add to precipitate phase for when order/disorder models are used
 
     def __init__(self, database, elements, phases, drivingForceMethod = 'approximate'):
-        self.db = Database(database)
+        if type(database) == str:
+            database = Database(database)
+        self.db = database
+        assert type(self.db) == Database  # make sure we have a pycalphad database at this point
```

Comment:
This assertion isn't necessary. If Database creation fails, an exception will be raised.

Reply:
Removed in the next commit.